client/app/billing/autoRenew/actions/update/form/form.constants.js

```diff
@@ -0,0 +1,8 @@
+export const AUTO_TYPE = 'automatic';
+
+export const RENEWAL_TYPES = [AUTO_TYPE, 'manualPayment'];
```

Comment:
I like to handle this case this way:
```suggestion
export const RENEWAL_TYPES = {
    automatic: 'automatic',
    manual: 'manualPayment',
};
```

This way you can `renewalType.type === RENEWAL_TYPES.automatic` or

````js
map(
    values(RENEWAL_TYPES),
    type => ({ type, label: this.$translate.instant(`billing_autorenew_service_update_service_${type}`) }),
)
````

Because you don't need `manualPayment ` but we might need it one day or the other